### DISCUSSION TITLE
Consigli Smart Home!

### DISCUSSION THREAD

**Discussion by AleParsifal** [indomus/forum#1268] [GENERALE - Discussioni generiche]
Salve a tutti. Io e la mia compagna ci siamo appena comprati casa, un appartamento da rifare completamente. In questi giorni stiamo facendo noi tutte le demolizioni, la seconda metà del mese invece dovrebbero partire le ristrutturazioni. Vorremo cogliere l’occasione per farci una casa “personalizzata”, soprattutto dal punto di vista dell’intrattenimento, motivo per il quale sono qua a chiedervi consiglio.

Innanzitutto il modem, dove piazzarlo? Vista la piantina pensavo di posizionarlo in alto (se non ho letto male è consigliato) di fronte alla porta del bagno, quindi in una zona più centrale possibile della casa. Non ci sono pareti portanti in cemento ma solo colonne portanti, per il resto tutte pareti da 10cm di foratini. Non sappiamo ancora se mantenere la Wind o passare ad un altro operatore (magari Fastweb), comunque l’idea sarebbe quella di comprare un modem terze parti, secondo voi l’intera casa verrebbe coperta con un buon segnale oppure sarà necessario usare dei ripetitori?

![Planimetria Ristrutturata (Con Misure)](https://user-images.githubusercontent.com/97262585/148466884-ddacde71-54ac-4ef8-80c9-1118c6af710d.png)

Ovviamente dovendo rifare l’intero impianto elettrico pensavamo (una volta scelta la postazione del modem) di far partire da quella posizione cavi eternet (pensavo Cat.6a) verso tutte le stanze. Qual è il miglior modo per far interfacciare il modem con tutti i vari cavi? Lo chiedo perché vedo che generalmente i modem hanno 4 uscite eternet mentre noi vorremo avere più di 4 uscite a muro nell’intera casa.

Per quanto riguarda il resto pensavamo di connettere alla rete un NAS, e poi eravamo interessati a tapparelle e termostato comandabili da remoto. Qualche consiglio in merito? Ci sono altre cose che varrebbe la pena aggiungere?

Grazie mille a tutti sin da ora per l’ottimo lavoro che fate!

**Comment by AleParsifal** [comment]
UP

**Reply by bircastri** [reply]
Ciao, il consiglio che ti do io è questo, parlo per gusti personali. Più cose puoi non mettere a vista meglio è sopratutto per se ti tratta del router.
Tieni presente che, io ho da poco installto la fibra della Telecom la fibra 1000 fino a casa. Loro ti mettono al posto dell intera placchetta, una placchetta orribile marcata Telecom tutta bianca che sporge dal muro, da li collegano la bredella ad un modem e da li con il cavo ethernet al router Telecom (qui puoi metterci ciò che vuoi).

Quindi tieni conto che dovresti avere 2 router più per avere la LAN in tutte le stanze ti servirà uno switch di rete. A questo punto io fossi in te farei questo:

Farei fare una predisposizione in ALTO nello sgabuzzino per la presa del "telefono" fibra quello che è, ti fai portare anche sempre in alto una 503 con due prese di corrente e li ci colleghi il tuo o i tuoi router. Poi sempre in alto una bella cassetta di derivazione GRANDE atta a contenere uno switch con almeno 8 porte. Tieni presente che lo switch va alimentato quindi ti serve pure la presa.

In sostanza dal router esce un cavon LAN che arriva allo switch e da questo poi derivi i cavi nelle varie stanze. Io fosse in te metterei un cavo LAN su ogni televisione, in più nella cameretta, dove va la scrivania un altra LAN poi sapete un pò voi le vostre esigenze.
Predisporrei anche una LAN nella SALA e se vedi che il wifi li non è forte ci metti un extender MESH (io ho quelli marcati TENDA NOVA), sono eccezionali e non hai più problemi di wifi o rete in casa

**Reply by AleParsifal** [reply]
> Ciao, il consiglio che ti do io è questo, parlo per gusti personali. Più cose puoi non mettere a vista meglio è sopratutto per se ti tratta del router. Tieni presente che, io ho da poco installto la fibra della Telecom la fibra 1000 fino a casa. Loro ti mettono al posto dell intera placchetta, una placchetta orribile marcata Telecom tutta bianca che sporge dal muro, da li collegano la bredella ad un modem e da li con il cavo ethernet al router Telecom (qui puoi metterci ciò che vuoi).
> 
> Quindi tieni conto che dovresti avere 2 router più per avere la LAN in tutte le stanze ti servirà uno switch di rete. A questo punto io fossi in te farei questo:
> 
> Farei fare una predisposizione in ALTO nello sgabuzzino per la presa del "telefono" fibra quello che è, ti fai portare anche sempre in alto una 503 con due prese di corrente e li ci colleghi il tuo o i tuoi router. Poi sempre in alto una bella cassetta di derivazione GRANDE atta a contenere uno switch con almeno 8 porte. Tieni presente che lo switch va alimentato quindi ti serve pure la presa.
> 
> In sostanza dal router esce un cavon LAN che arriva allo switch e da questo poi derivi i cavi nelle varie stanze. Io fosse in te metterei un cavo LAN su ogni televisione, in più nella cameretta, dove va la scrivania un altra LAN poi sapete un pò voi le vostre esigenze. Predisporrei anche una LAN nella SALA e se vedi che il wifi li non è forte ci metti un extender MESH (io ho quelli marcati TENDA NOVA), sono eccezionali e non hai più problemi di wifi o rete in casa

Ciao,
grazie per la risposta!

Leggendo un po’ di post mi sa che mi sono istruito per bene perché in pratica è proprio ciò che ho fatto.

Nel disimpegno, di fronte alla porta del bagno, c’è una nicchia in cui metteremo un piccolo armadio che faccia da porta giacchetti, al di sopra di quello, a circa 2 metri e 30 dal pavimento abbiamo fatto predisporre tutto. 2 scatoline 503 per la corrente, l’arrivo di internet in casa per collegare il modem wifi, e poi una cassetta in cui confluiscono tutti i cavi ethernet che abbiamo disseminato in giro per casa che collegheremo allo switch che andrà collegato a sua volta al modem. Abbiamo la possibilità di mettere la fibra 1000 di open fiber mantenendo la Wind, oppure scegliendo Vodafone, Iliad o uno di vari operatori che non ho mai sentito nominare. Quindi in fase di trasloco chiederò espressamente di voler utilizzare un modem terze parti per essere autonomo.

Nel salotto, nell’angolo in basso a destra guardando la piantina, ho fatto mettere un punto in cui eventualmente collegare un ripetitore per aumentare il segnale wifi.

Inoltre per controllare da remoto le tapparelle ho comprato dei relè della LoraTap da inserire nelle scatoline 503 dice verranno installati i pulsanti delle tapparelle.

A questo punto non resta che comprare i vari dispositivi, solo che fra tutti i modelli che sto trovando online non ho idea quale comprare. Senza esagerare con i prezzi però un po’ inizialmente pensavo di investirci, così da non doverci rimettere le mani dopo poco.
Quindi ti chiedo, consigli su cosa acquistare come:
-modem wifi
-switch
-ripetitore
-nas
-hdd per nas
-termostato

Grazie ancora!
A presto

**Reply by indomus** [reply]
Ciao,
purtroppo non è possibile andare sul link specifici, per regolamento. Quel che si può fare è referenziare quanto si è testato su inDomus: sui modem, valuta [FRITZ!Box](https://indomus.it/componenti/avm-linea-modem-router-fritzbox/), sui termostati valuta [Tado](https://indomus.it/componenti/termostato-intelligente-tado/) sicuramente, sulle NAS valuta [QNAP](https://indomus.it/category/componenti/?s=qnap).

**Reply by bircastri** [reply]
Io come router e ripetitori ti consiglio di valutare la marca NOVA su Amazon li trovi ma si trovano anche sui siti cinesi. Io li ho installati da 2 anni, e li ho consigliati ad un amico (che ha una casa a 4 piani ) e da quando li utilizza ha risolto tutti i suoi problemi. Secondo me sono un ottimo rapporto qualità prezzo.
Il Fritz che ti è stato suggerito senza dubbio oggi è (da quello che so), la ferrari dei ruoter domestici.....

Per il termostato io ho quello della Btcino, ma solo perchè mi piaceva di più l'estetica rispetto al Tado che so (parlando un amico che ha dei B&B) un ottimo e validissimo prodotto. 

A questo punto però visto che ti trovi nel ballo, e metti le tapparelle elettriche domotiche, perchè non domotizzi le luci, prese degli elettrodomestici ? (sopratutto forno, lavatrice, lavastoviglie, asciugatrice) ?

**Reply by AleParsifal** [reply]
> Ciao,
> purtroppo non è possibile andare sul link specifici, per regolamento. Quel che si può fare è referenziare quanto si è testato su inDomus: sui modem, valuta [FRITZ!Box](https://indomus.it/componenti/avm-linea-modem-router-fritzbox/), sui termostati valuta [Tado](https://indomus.it/componenti/termostato-intelligente-tado/) sicuramente, sulle NAS valuta [QNAP](https://indomus.it/category/componenti/?s=qnap).

Grazie mille, dò subito un’occhiata ai link!

**Reply by AleParsifal** [reply]
> Io come router e ripetitori ti consiglio di valutare la marca NOVA su Amazon li trovi ma si trovano anche sui siti cinesi. Io li ho installati da 2 anni, e li ho consigliati ad un amico (che ha una casa a 4 piani ) e da quando li utilizza ha risolto tutti i suoi problemi. Secondo me sono un ottimo rapporto qualità prezzo. Il Fritz che ti è stato suggerito senza dubbio oggi è (da quello che so), la ferrari dei ruoter domestici.....
> 
> Per il termostato io ho quello della Btcino, ma solo perchè mi piaceva di più l'estetica rispetto al Tado che so (parlando un amico che ha dei B&B) un ottimo e validissimo prodotto.
> 
> A questo punto però visto che ti trovi nel ballo, e metti le tapparelle elettriche domotiche, perchè non domotizzi le luci, prese degli elettrodomestici ? (sopratutto forno, lavatrice, lavastoviglie, asciugatrice) ?

Grazie mille, di nuovo, per i consigli. Ma scusami l’ignoranza, per le luci ok, ma per gli elettrodomestici che funzioni potrei guadagnare?

**Reply by bircastri** [reply]
beh potresti ad esempio conoscere lo stato della lavatrice se è in funzione o meno, potresti creare una scena per cui se se vedi che i tuoi elettrodomestici + luci stanno consumando Xkw e tu hai un contatore da 3KW piuttosto che dover scendere per le scale e riattivare il tuo contatore, magari automaticamente il sistema ti manda una notifica, ti fai avvisare tramite alexa o in un altro modo...

in più in un sistema domotico più complesso quale potrebbe essere Home Assistant potresti monitorare tutti i tuoi consumi (ma questa è roba da nerd)

**Reply by indomus** [reply]
Tipo [così](https://indomus.it/progetti/dedurre-lo-stato-di-un-elettrodomestico-non-domotico-su-home-assistant-con-sonoff-pow-o-similari/).

**Reply by AleParsifal** [reply]
> beh potresti ad esempio conoscere lo stato della lavatrice se è in funzione o meno, potresti creare una scena per cui se se vedi che i tuoi elettrodomestici + luci stanno consumando Xkw e tu hai un contatore da 3KW piuttosto che dover scendere per le scale e riattivare il tuo contatore, magari automaticamente il sistema ti manda una notifica, ti fai avvisare tramite alexa o in un altro modo...
> 
> in più in un sistema domotico più complesso quale potrebbe essere Home Assistant potresti monitorare tutti i tuoi consumi (ma questa è roba da nerd)



> Tipo [così](https://indomus.it/progetti/dedurre-lo-stato-di-un-elettrodomestico-non-domotico-su-home-assistant-con-sonoff-pow-o-similari/).



E in quanto nerd queste cose mi piacciono un sacco 🤣! 

Grazie mille a entrambi però consigli!!
